### ISSUE TITLE
Example command given in `generate-release` docs gives a "migration guide not found! warning"

### ISSUE THREAD

**Issue by alice-i-cecile** [bevyengine/bevy-website#1190]
```
PS C:\Users\alice\Documents\bevy-website\generate-release> cargo run -- --from v0.13.0 --to main --release-version 0.14 migration-guides
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.12s
     Running `C:\Users\alice\Documents\bevy-website\target\debug\generate-release.exe --from v0.13.0 --to main --release-version 0.14 migration-guides`
Getting list of all commits from v0.13.0 to main
Page: 1 (250 commits)
Page: 2 (250 commits)
Page: 3 (199 commits)
Page: 4 (0 commits)
Found 699 commits
Getting list of all merged PRs from v0.13.0 to main with label None
Page: 1 (65 prs)
Page: 2 (58 prs)
Page: 3 (66 prs)
Page: 4 (68 prs)
Page: 5 (65 prs)
Page: 6 (56 prs)
Page: 7 (58 prs)
Page: 8 (68 prs)
Page: 9 (60 prs)
Page: 10 (64 prs)
Page: 11 (53 prs)
Page: 12 (37 prs)
Skipping PR closed before the target datetime 2023-10-18 17:55:10 UTC
Page: 13 (14 prs)
Page: 14 (3 prs)
Skipping PR closed before the target datetime 2024-02-02 21:34:13 UTC
Page: 15 (0 prs)
Found 709 merged PRs and 699 commits since 2024-02-17 (the base commit date)

Found 133 breaking PRs merged
migration guide not found!
```
No other instructions of required steps were present. This doesn't actually seem to fail: the content is still generated in the `release-content` folder.

**Comment by alice-i-cecile** [comment]
The same error was *not* found for the release-notes variant!

**Comment by alice-i-cecile** [comment]
Interesting: the `release-notes` variant made an empty `migration-guides` folder for me. I bet that's why this was missed.

**Comment by alice-i-cecile** [comment]
This is partly my fault: I didn't make the stub file as laid out in the README. However! The error should have told me I was being dumb and to check the README.

**Comment by TrialDragon** [comment]
Hmm, that warning is confusing. Digging into the tool's code, it seems to be actually trying to say "This PR, marked as breaking changes, doesn't have a migration guide section in it's body of text to get relevant content from; outputting default TODO to file instead." It's not too terribly concerning, more just a notification that hey, we have to write something ourselves instead.

**Comment by alice-i-cecile** [comment]
Ah. Okay, that should definitely list the PR number in the log then.

**Comment by TrialDragon** [comment]
Hmm, yeah. The function would need to be changed; it currently doesn't have access to that info. (Also, why is that it's own file / module I wonder; it is solely used by the migration guides and clearly has code designed with the migration guides in mind).

**Comment by IceSentry** [comment]
Oh, I think I know what happened. I removed a ton of logging. Previously it would list each PR it was processing so it was obvious which one it was about. I forgot to update the message when removing those logs.

**Comment by IceSentry** [comment]
> Also, why is that in its own file / module I wonder; it is solely used by the migration guides and clearly has code designed with the migration guides in mind

Technically, none of the code is migration guide specific. It looks for a section header and prints the content in a consistent way. The only migration guide reference is in the comment and I only phrased it like that because when I added that comment I was thinking about migration guides. There's not really anything in there that is migration guide specific. At some point it was also used for the changelogs to write the `added/changed` thing, but we never used it so I removed it.
